src/components/Article/Article.jsx

```diff
@@ -1,6 +1,68 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Article.css';
 
-// Create an Article function returning the HTML of article block
+function Article() {
+  return (
+    <article className="article">
+      <h1 className="article__title">Headline</h1>
+      <p className="article__paragraph">
+        In elementum lorem eget est euismod ornare. Phasellus sit amet
+        pellentesque mauris. Aliquam quis malesuada ex. Nullam eu aliquam
+        nibh. Mauris molestie, urna accumsan ornare semper, augue nibh
+        posuere lorem, vitae feugiat sem magna eget massa. Vivamus quis
+        tincidunt dolor. Fusce efficitur, orci non vestibulum consequat,
+        lectus turpis bibendum odio, in efficitur leo felis sed justo. Fusce
+        commodo iaculis orci, quis imperdiet urna. Sed mollis facilisis lacus
+        non condimentum. Nunc efficitur massa non neque elementum semper.
+        Vestibulum lorem arcu, tincidunt in quam et, feugiat venenatis augue.
+        Donec sed tincidunt tellus, a facilisis magna. Proin sit amet viverra
+        nibh, bibendum gravida felis. Vivamus ut nunc id mauris posuere
+        pellentesque. Praesent tincidunt id odio id feugiat.
+      </p>
+      <p className="article__paragraph">
```

Comment:
```suggestion
      <h1 className="article__title">Headline</h1>
      
      <p className="article__paragraph">
        In elementum lorem eget est euismod ornare. Phasellus sit amet
        pellentesque mauris. Aliquam quis malesuada ex. Nullam eu aliquam
        nibh. Mauris molestie, urna accumsan ornare semper, augue nibh
        posuere lorem, vitae feugiat sem magna eget massa. Vivamus quis
        tincidunt dolor. Fusce efficitur, orci non vestibulum consequat,
        lectus turpis bibendum odio, in efficitur leo felis sed justo. Fusce
        commodo iaculis orci, quis imperdiet urna. Sed mollis facilisis lacus
        non condimentum. Nunc efficitur massa non neque elementum semper.
        Vestibulum lorem arcu, tincidunt in quam et, feugiat venenatis augue.
        Donec sed tincidunt tellus, a facilisis magna. Proin sit amet viverra
        nibh, bibendum gravida felis. Vivamus ut nunc id mauris posuere
        pellentesque. Praesent tincidunt id odio id feugiat.
      </p>
      
      <p className="article__paragraph">
```

Add spaces between nested siblings.